ra/ra.go

```diff
@@ -25,6 +25,7 @@ import (
 	bgrpc "github.com/letsencrypt/boulder/grpc"
 	blog "github.com/letsencrypt/boulder/log"
 	"github.com/letsencrypt/boulder/metrics"
+	"github.com/letsencrypt/boulder/policy"
```

Comment:
Instead of introducing a cross-package dependency, let's just give `ExtractDomainIANASuffix` its own package - maybe `iana.ExtractSuffix`?